businessRules/ideas.js

```diff
@@ -71,7 +62,7 @@ function approvedIdeasList() {
     return approvedIdeas;
 }
 
-function rejectedIdeasList() {
+function getRejectedIdeasList() {
```

Comment:
Or simply: `getRejectedIdeas`

The word `list` is redundant in naming variables, because any data that is multiple items, most likely an array

Reply:
To specific, understood that. Fixed